.buildkite/scripts/steps/ess.ps1

```diff
@@ -1,10 +1,11 @@
 function ess_up {
   param (
       [string]$StackVersion,
+      [string]$StableSnapshotVersion,
       [string]$EssRegion = "gcp-us-west2"
   )
 
-  Write-Output "~~~ Starting ESS Stack"
+  Write-Output "~~~ Starting ESS. Version: $StackVersion. Stable snapshot: $StableSnapshotVersion"
```

Comment:
I'm not much familiar with the ITs, but I cannot see this trace in one of the Windows ITs:
- https://buildkite.com/elastic/elastic-agent/builds/20447#01966817-7b5b-4f0a-8cb4-56fe4bda1983/1745503548222/Running%20global%20environment%20hook


<img width="1352" alt="image" src="https://github.com/user-attachments/assets/1f776e45-f6bb-442d-af57-67e23a35d820" />